packages/tatum-doge/src/index.ts

```diff
@@ -0,0 +1,9 @@
+import 'reflect-metadata';
+
+export * from './wallet';
+export * from './model';
+export * from './ledger';
+export * from './offchain';
+export * from './blockchain';
+export * from './transaction';
+export * from './constants'
```

Comment:
I guess it would be great also re-export other modules from tatum-core. See e.g. `tatum-eth/src/index.ts`.